clickhouse_driver/columns/ipcolumn.py

```diff
@@ -0,0 +1,64 @@
+from ipaddress import IPv4Address, IPv6Address, AddressValueError
+
+from .. import errors
+from ..util import compat
+from .exceptions import ColumnTypeMismatchException
+from .stringcolumn import ByteFixedString
+from .intcolumn import UInt32Column
+
+
+class IPv4(UInt32Column):
+    ch_type = "IPv4"
+    py_types = compat.string_types + (IPv4Address, int)
+
+    def check_item_type(self, value):
+        if not isinstance(value, self.py_types):
+            raise ColumnTypeMismatchException(value)
+
+    def before_write_item(self, value):
+        # allow Ipv4 in integer, string or IPv4Address object
+        try:
+            if isinstance(value, int):
+                return value
+
+            if not isinstance(value, IPv4Address):
+                value = IPv4Address(value)
+
+            return int(value)
+        except AddressValueError:
+            raise errors.CannotParseDomainError(
+                "Cannot parse IPv4 '{}'".format(value)
+            )
+
+    def after_read_item(self, value):
+        return IPv4Address(value)
+
+
+class IPv6(ByteFixedString):
+    ch_type = "IPv6"
+    py_types = IPv6Address, bytes
```

Comment:
`py_types = compat.string_types + (IPv6Address, bytes)`?

Reply:
My mistake, fixed (the test now enforce the type check to ensure the type is in py_types)